### PR TITLE
QP-1722: Part 2 Updated Kind Functionality

### PR DESCRIPTION
**Pull request by bzvestey**
https://maanainc.atlassian.net/browse/QP-1722

* Added some comments
* Kind of Service by Name

Screen Shot 2020-10-07 at 3 32 45 PM
Screen Shot 2020-10-07 at 3 31 51 PM
Screen Shot 2020-10-07 at 3 30 34 PM

### CODE REVIEW

File: src/AssistantAPIClient.js [hunk 1]
```diff
@@ -291,17 +291,105 @@ class AssistantAPIClient {
...
-  getKindById = id => APICall('getKindById', id)
+  /**
+   * Returns a Kind with the given name from a specific service.
+   *
+   * @param {string} serviceId ID of the service the function lives in.
+   * @param {string} name The name of the Kind to find.
+   *
+   * @returns {Promise<Kind>} The requested Kind.
+   */
+  getKindOfServiceByName(serviceId, name) {
```
Comment (mechaffin) [inline src/AssistantAPIClient.js:367]: Same from/of comment as from the other review.  The similar 'function'  functions use 'from'. Be consistent.

File: src/AssistantAPIClient.js [hunk 2]
```diff
@@ -291,17 +291,105 @@ class AssistantAPIClient {
...
-  getKindsById = ids => APICall('getKindsById', ids)
+  /**
+   * Returns a list of Kinds with the given names from a specific service.
+   *
+   * @param {string} serviceId ID of the service the function lives in.
+   * @param {Array<string>} names The names of the Kinds to find.
+   *
+   * @returns {Promise<Array<Kind>>} The list of requested Kinds.
+   */
+  getKindsOfServiceByName(serviceId, names) {
```
Comment (mechaffin) [inline src/AssistantAPIClient.js:379]: from/of.

**Review by mechaffin — CHANGES_REQUESTED** [review]

File: src/AssistantAPIClient.js (hunk 1 above)
Comment (bzvestey) [reply src/AssistantAPIClient.js:367]: Based on our conversation I updated other places to use this pattern.

Comment (bzvestey) [reply src/AssistantAPIClient.js:367]: This ended up needing no changes to the assistant API its self.

**Review by mechaffin — APPROVED** [review]

**Pull request merged**